org.mixedrealitytoolkit.standardassets/CHANGELOG.md

```diff
@@ -4,7 +4,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [3.1.0-development] - 2024-02-09
 
+### Added
+
+* Added SimpleEmptyButton (Experimental) and SimpleActionButton (Experimental) prefabs plus corresponding (Experimental) assets and demoscene. (#635)
+
 ### Fixed
 
-* Fixed support for UPM package publishing in the Unity Asset Store.
-* Added SimpleEmptyButton (Experimental) and SimpleActionButton (Experimental) prefabs plus corresponding (Experimental) assets and demoscene.
+* Fixed support for UPM package publishing in the Unity Asset Store. (#519)
```

Comment:
You need do add the full URL for the PRs....The hashtags don't function outside of github.  For example

([#519](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/519))